### PR TITLE
[CI] Fix golang unit tests

### PR DESCRIPTION
**Pull request by alphaX86**
Signed-off-by: Aadhitya A <aadhitya864@gmail.com>

**Description**

This PR fixes #

**Notes for Reviewers**
WIP till all CI tests get resolved

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by alphaX86** [comment]
Errors settled:

- [ ] offboard_test.go
- [ ] onboard_test.go
- [ ] Others (if any)

File: mesheryctl/internal/cli/root/app/onboard_test.go [hunk 1]
```diff
@@ -62,6 +61,11 @@ func TestOnboardCmd(t *testing.T) {
 			Args:             []string{"onboard", "-f", filepath.Join(fixturesDir, "sampleApp.golden"), "--skip-save"},
 			ExpectedResponse: "onboard.output.golden",
 			URLs: []utils.MockURL{
+				{
+					Method:       "POST",
```
Comment (leecalcote) [inline mesheryctl/internal/cli/root/app/onboard_test.go:65]: Is this posting the "sampleApp.golden"?

**Review by leecalcote — APPROVED** [review]

**Comment by alphaX86** [comment]
All test errors resolved

**Comment by l5io** [comment]
Task linked: [CU-2k08ra9 Fix unit test bug](https://app.clickup.com/t/2k08ra9)

File: mesheryctl/pkg/utils/helpers.go [hunk 2]
```diff
@@ -794,9 +794,6 @@ func SetOverrideValues(ctx *config.Context, mesheryImageVersion string) map[stri
 		"meshery-traefik-mesh": map[string]interface{}{
 			"enabled": false,
 		},
-		"meshery-cpx": map[string]interface{}{
-			"enabled": false,
```
Comment (leecalcote) [inline mesheryctl/pkg/utils/helpers.go:798]: Thank you. // @hershd23

**Review by leecalcote — APPROVED** [review]
LGTM

**Pull request merged**